From df68517bb56472bc39d219bab2bafbe43116a1ed Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 10 Jan 2018 21:33:51 +0100
Subject: [PATCH 1/1] configs: sunxi: enable BLK, DM_MMC for Bananapi

We want to move the device driver model. So make the switch for
the Bananapi block drivers.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 configs/Bananapi_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/Bananapi_defconfig b/configs/Bananapi_defconfig
index 52650448091..300110a5cab 100644
--- a/configs/Bananapi_defconfig
+++ b/configs/Bananapi_defconfig
@@ -17,6 +17,8 @@ CONFIG_SPL_I2C_SUPPORT=y
 # CONFIG_SPL_EFI_PARTITION is not set
 CONFIG_NETCONSOLE=y
 CONFIG_SCSI_AHCI=y
+CONFIG_BLK=y
+CONFIG_DM_MMC=y
 CONFIG_ETH_DESIGNWARE=y
 CONFIG_RGMII=y
 CONFIG_SUN7I_GMAC=y
-- 
2.11.0

